{"name": "# C5163 Device Locations section", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5163_Device_Locations_section.py\", line 8, in dvc_lctns_txt_hr\n    context.app.main_page.dvc_lctns_txt_hr(dvc_lctns_txt_hr)\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UIautomation_python\\pages\\main_page.py\", line 975, in dvc_lctns_txt_hr\n    dvc_lctns_txt_hr = wait.until(EC.visibility_of_element_located(DVC_LCTNS_TXT_HR)).text\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Loginpage", "status": "passed", "start": 1605494907222, "stop": 1605494910023}, {"name": "Then Login with the given credentials", "status": "passed", "start": 1605494910023, "stop": 1605494912913}, {"name": "Then Click Dashboard menu icon and make sure Dashboard page reloaded", "status": "passed", "start": 1605494912913, "stop": 1605494913087}, {"name": "And Make sure Device Locations words are on the screen", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5163_Device_Locations_section.py\", line 8, in dvc_lctns_txt_hr\n    context.app.main_page.dvc_lctns_txt_hr(dvc_lctns_txt_hr)\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UIautomation_python\\pages\\main_page.py\", line 975, in dvc_lctns_txt_hr\n    dvc_lctns_txt_hr = wait.until(EC.visibility_of_element_located(DVC_LCTNS_TXT_HR)).text\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1605494913088, "stop": 1605494931357}, {"name": "And Mouse hover question mark in top right of the block and make sure tooltip appears", "status": "skipped", "start": 1605494933480, "stop": 1605494933480}, {"name": "Then Make sure that click on Device Locations drop-down list is working", "status": "skipped", "start": 1605494933480, "stop": 1605494933480}], "start": 1605494901915, "stop": 1605494933480, "uuid": "e2783cf8-6d66-4013-8696-356b828ff197", "historyId": "b89a768134587347e4e0ce9a72fe873a", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "# C5163 Device Locations section"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}